.changeset/dry-frogs-smash.md

```diff
@@ -4,4 +4,4 @@
 
 `hds-tooltip` - Changed structure of tooltip content to add a wrapper that is always in the DOM and set `aria-controls` on trigger elements for a11y improvements with toggled content
 
-`TooltipButton` - Changed structure of tooltip content to add a wrapper that is always in the DOM and set `aria-controls` on button for a11y improvements with toggled content
+`Tooltip` - Changed structure of tooltip content to add a wrapper that is always in the DOM and set `aria-controls` on button for a11y improvements with toggled content
```

Comment:
Although in code it is `TooltipButton`, we refer to it as Tooltip in [website](https://helios.hashicorp.design/components/tooltip) (including the directory structure.